packages/styles/generic/_g.reset.scss

```diff
@@ -0,0 +1,142 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+button,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
```

Comment:
Hello,

Quel est l'intérêt de mettre une propriété `margin:0;` sur des éléments qui n'ont pas cette propriété par défaut ?
Idem pour le padding.
Concernant le `border:0;`, est-on conscient qu'il va générer sur toutes ces balises ceci côté browser ? : 
```
border-bottom-color:rgb(0, 0, 0);
border-bottom-style:none;
border-bottom-width:0px;
border-image-outset:0px;
border-image-repeat:stretch;
border-image-slice:100%;
border-image-source:none;
border-image-width:1;
border-left-color:rgb(0, 0, 0);
border-left-style:none;
border-left-width:0px;
border-right-color:rgb(0, 0, 0);
border-right-style:none;
border-right-width:0px;
border-top-color:rgb(0, 0, 0);
border-top-style:none;
border-top-width:0px;
```

Reply:
Hello, Gael,
The part you just commented so vehemently is just a copy/paste of the existing Core Team side.

So, I suggest you discuss it with them to know how much the reset embedded in Mozaic must be different from the existing one, and if they are able to absorb the possible impacts ;)

Reply:
Hi,

Sorry for my bad explanation.

In fact, about comment above, to be more understanding, maybe it will be better if we did this :
```scss
html {
  @include set-font-face();
  box-sizing: border-box;
}

body {
  line-height: 1.5;
  scroll-behavior: smooth;
}

body,
blockquote,
figure {
  margin: 0;
}

h1,
h2,
h3,
h4,
h5,
h6 {
  font-size: 100%;
  font-weight: normal;
}

h1,
h2,
h3,
h4,
h5,
h6,
p,
ol,
ul,
dl,
dt,
textarea {
  margin-top: 0;
  margin-bottom: 0;
}

ol,
ul,
dd,
fieldset {
  margin-left: 0;
}

fieldset {
  margin-right: 0;
}

ol,
ul {
  padding-left: 0;
}

img,
video {
  height: auto;
  max-width: 100%;
}

/* table, inputs, etc.. missing */
```

Why ? Because shortcuts are good but only when they are jutified.

With these screenshots, we can see the difference of the calculated styles (Each calculated style uses CPU)

With shortcuts : 
![image](https://user-images.githubusercontent.com/3939846/81916470-271c6880-95d4-11ea-8d9e-8ee214c7b3b3.png)

Without shortcuts : 
![image](https://user-images.githubusercontent.com/3939846/81916913-c6416000-95d4-11ea-94f1-fd4e1a111555.png)

If we force a `margin-left:0;` for an element with a default `margin:0;`, I think it will use useless CPU and impact performance.

Do we know if shortcuts (in this case) will not have an impact on performance ?

We can discuss about this on google meet, maybe it will be more understanding.

Reply:
Hi Gael,
Thank you for your comment.
Personally, I don't know enough about the impact of CSS on the CPU, but your idea of only overloading what is necessary seems good to me.

I have just made the requested adjustments.
I'll let you check it out and tell me if it's better this way.